core/src/components/breadcrumbs/test/basic/index.html

```diff
@@ -0,0 +1,238 @@
+          Data
+        </ion-breadcrumb>
+      </ion-breadcrumbs>
+
+
+      <p>Breadcrumbs (Secondary); Breadcrumb (Danger)</p>
+
+      <ion-breadcrumbs color="secondary">
+        <ion-breadcrumb href="#">
+          Home
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#library" color="danger">
+          Library
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#disabled" disabled>
+          Disabled
+        </ion-breadcrumb>
+        <ion-breadcrumb>
+          Data
+        </ion-breadcrumb>
+      </ion-breadcrumbs>
+
+      <p>Custom Separator: Icon w/ last separator showing</p>
+
+      <ion-breadcrumbs>
+        <ion-breadcrumb href="#">
+          Home
+          <ion-icon slot="separator" name="caret-forward-outline"></ion-icon>
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#library">
+          Library
+          <ion-icon slot="separator" name="caret-forward-outline"></ion-icon>
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#disabled" disabled>
+          Disabled
+          <ion-icon slot="separator" name="caret-forward-outline"></ion-icon>
+        </ion-breadcrumb>
+        <ion-breadcrumb separator="true">
+          Data
+          <ion-icon slot="separator" name="star-outline"></ion-icon>
+        </ion-breadcrumb>
+      </ion-breadcrumbs>
+
+      <p>Breadcrumbs: default wrapping</p>
+
+      <ion-breadcrumbs>
+        <ion-breadcrumb href="#">
+          Home
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#library">
+          Cameras & Camcorders
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#digital-camera-accessories">
+          Digital Camera Accessories
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#camera-lenses">
+          Camera Lenses
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#dslr-lenses">
+          DSLR Lenses
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#prime-lenses">
+          Prime Lenses
+        </ion-breadcrumb>
+        <ion-breadcrumb>
+          Product Info
+        </ion-breadcrumb>
+      </ion-breadcrumbs>
+
+      <p>Breadcrumbs: collapsed</p>
+
+      <ion-breadcrumbs>
+        <ion-breadcrumb href="#" collapsed>
+          Home
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#library" collapsed>
+          Cameras & Camcorders
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#digital-camera-accessories" collapsed>
+          Digital Camera Accessories
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#camera-lenses">
+          Camera Lenses
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#dslr-lenses">
+          DSLR Lenses
+        </ion-breadcrumb>
+        <ion-breadcrumb href="#prime-lenses">
+          Prime Lenses
+        </ion-breadcrumb>
+        <ion-breadcrumb>
+          Product Info
+        </ion-breadcrumb>
+      </ion-breadcrumbs>
+
+      <p>Breadcrumbs: collapsed middle</p>
+
+      <ion-breadcrumbs>
```

Comment:
When I set `color="danger"` on `ion-breadcrumbs`, it only gets applied to "Product Info", not "Home". Is this expected? When I set `color="danger"` on the `ion-breadcrumb` for "Home", it works as expected.

Reply:
@liamdebeasi Yeah Ben wanted it to only change the color of the active breadcrumb. But if you apply it to individual breadcrumbs it would change their color.